py3status/modules/wifi.py

```diff
@@ -66,16 +68,29 @@ class Py3status:
     cache_timeout = 10
     device = 'wlan0'
     down_color = 'bad'
-    format_down = 'W: down'
-    format_up = 'W: {bitrate} {signal_percent} {ssid}'
+    format = DEFAULT_FORMAT
     round_bitrate = True
     signal_bad = 29
     signal_degraded = 49
     use_sudo = False
 
     def post_config_hook(self):
-        self._ssid = None
-        self._max_bitrate = 0
```

Comment:
Yes, I think so as I didn't run into this issue on my laptop, only netbook... And I'd like better wifi device autodetection. No more inputting device manually. Really, we should look through everything with this module like we did with dead deadbeef.